src/components/StatusButton.tsx

```diff
@@ -79,9 +88,12 @@ const StatusButton = (props: Props) => {
       </Status>
       <ButtonContainer>
         <Button
+          style={{ minWidth: '4.3rem' }}
           small
           onPress={onPress}
-          disabled={status === CompletionStatus.Complete || disabled}
+          disabled={
+            (status === CompletionStatus.Complete && !enabled) || disabled
```

Comment:
Not my favorite way but this ensures that enabled just overrides the status buttons default behavior.